api/tests/routes/native/v1/offers_test.py

```diff
@@ -225,21 +236,41 @@ def test_get_digital_offer_without_available_activation_code(self, app):
 
         queries = 1  # select offer
         queries += 1  # get available_activation_code for each offer.stocks
+        queries += 1  # select feature
 
         # when
-        with assert_num_queries(2):
+        with assert_num_queries(queries):
             response = TestClient(app.test_client()).get(f"/native/v1/offer/{offer_id}")
 
         # then
         assert response.status_code == 200
         assert response.json["stocks"][0]["activationCode"] is None
 
+    @patch("pcapi.core.booking_providers.cds.client.CineDigitalServiceAPI.get_shows_remaining_places")
+    @override_features(ENABLE_CDS_IMPLEMENTATION=True)
+    def test_get_offer_from_cds_with_no_remaining_places(self, mocked_get_shows_remaining_places: Any, app) -> None:
+        venue = VenueFactory()
+        VenueBookingProviderFactory(venue=venue)
+        offer = OfferFactory(subcategoryId=subcategories.SEANCE_CINE.id, name="test_offre", venue=venue)
+        stock = EventStockFactory(offer=offer, price=5, quantity=10, idAtProviders="1")
+
+        mocked_get_shows_remaining_places.return_value = {1: 0}
+
+        TestClient(app.test_client()).get(f"/native/v1/offer/{offer.id}")
+
+        mocked_get_shows_remaining_places.assert_called_once()
+        offer_stock = Stock.query.filter(Stock.id == stock.id).first()
+
+        assert offer_stock.quantity == 0
```

Comment:
```suggestion
        assert offer_stock.quantity == 0
        assert response.json["stocks"][0]["isSoldOut"]
```
Je pense ça peut être mieux sinon on teste aussi que le stock renvoyé est épuisé.